src/utils/gitlab/converter.ts

```diff
@@ -0,0 +1,449 @@
+//#region > Imports
+//#PACKAGE "moment"
+//## npm install "moment"@2.25.3
+// A lightweight JavaScript date library for parsing,
+// validating, manipulating, and formatting dates.
+import moment from "moment";
+
+//> Models
+// Contains all reducer database models
+import * as models from "../../reducer/database/models";
+//#endregion
+
+//#region > Interfaces
+/**
+ * @interfaces ScrapedData defines the data structure required
+ *             for the converter.
+ */
+interface IScrapedData {
+  /** Manually set platform information. */
+  platform: {
+    url: string;
+    name: string;
+  };
+  /** DOM data of a users profile page. */
+  home: Document;
+  /**
+   * DOM data of a users news feed. Contains
+   * less contribution information than activity.
+   */
+  atom?: Document;
+  /**
+   *  Contribution calendar of a user. Contains only days of the current year.
+   */
+  currentCalendar: { [index: string]: number };
+  /**
+   *  JSON data of a users groups.
+   */
+  groups: {
+    html: string;
+  };
+  /**
+   *  JSON data of a projects/repositories groups.
+   */
+  projects: {
+    html: string;
+  };
+  /**
+   *  JSON data of a users activity feed. Contains information about every
+   *  contribution.
+   */
+  activity: {
+    html: string;
+  };
+}
+
+/** @interfaces CalendarDays defines the data structure for the calendar */
+interface CalendarDays {
+  [date: string]: {
+    contributions: {
+      [type: string]: {
+        [datetime: string]: { total: number; name: string; repoUrl: string };
+      };
+    };
+    total: number;
+  };
+}
+
+/** @interfaces Statistic defines the data structure for a statistic year */
+interface Statistic {
+  [year: number]: {
+    [type: string]: {
+      total: number;
+    };
+  };
+}
+//#endregion
+
+//#region > Functions
+/**
+ * Converter for data from the scraper.
+ *
+ * @function
+ * @param rawData Data to be processed. Must fit IScrapedData format
+ * @description Fill the database with data provided by "rawData"
+ */
+function runScraper(rawData: IScrapedData) {
+  /* Extract platform data from rawData.home document */
+  let platformName = rawData.platform.name;
+
+  let platformUrl = rawData.platform.url;
```

Comment:
Is this newline required?
```suggestion
  let platformName = rawData.platform.name;
  let platformUrl = rawData.platform.url;
```